src/test/java/net/conjur/api/ConjurTest.java

```diff
@@ -23,7 +23,7 @@
  */
 public class ConjurTest {
 
-    private static final String VARIABLE_KEY = "test/testVariable";
+    private static final String VARIABLE_KEY = "testVariable";
```

Comment:
I think this really needed to be "test/testVariable" - I tried running tests on this branch as-is and it fails, but changing this back to "test/testVariable" they succeed.

Reply:
In order it to work it requires adding a policy with the testVariable value.
First, the comment consist of a mistake. It doesn't match the VARIABLE_KEY value.
Second, the initial test/ is useless.
I did checked it here before applying for the PR and it works properly.